guide/sections/global-services.adoc

```diff
@@ -117,3 +121,16 @@ TODO: to be completed
 Procedures pertinent to the Global Monitor will be described here once validated through testing during the WIS2 pilot phase.
 
 TODO: to be completed
+
+=== Global Registry ===
+
+* The role of the Global Registry is to make available the information required for Global Services and WIS nodes to connect with each other. 
+* The Global Registry makes available to Global Brokers the list of registered WIS Nodes. The following metadata is made available for each WIS Node: hostname and port(s) of local broker, center_id, list of implemented protocols (MQTT, MQTTs, web-socket).
+* The Global Registry issues X.509 certificates to all GBs and GCs and periodically renews them.
```

Comment:
Same as above. I really don't think we should get into this. Transforming the Registry into a CA seems an extremely challenging work.